### ISSUE TITLE
Transient failure on RemoteTaskRunnerTest.testRun

### ISSUE THREAD

**Issue by gianm** [apache/incubator-druid#2199]
https://travis-ci.org/druid-io/druid/jobs/100115132

```
testRun(io.druid.indexing.overlord.RemoteTaskRunnerTest)  Time elapsed: 3.318 sec  <<< FAILURE!
java.lang.AssertionError: expected:<SUCCESS> but was:<FAILED>
    at org.junit.Assert.fail(Assert.java:88)
    at org.junit.Assert.failNotEquals(Assert.java:743)
    at org.junit.Assert.assertEquals(Assert.java:118)
    at org.junit.Assert.assertEquals(Assert.java:144)
    at io.druid.indexing.overlord.RemoteTaskRunnerTest.testRun(RemoteTaskRunnerTest.java:131)
```

**Comment by fjy** [comment]
Fixed by #2201 (maybe?)

**Comment by fjy** [comment]
Reopening, as #2201 did not fix things
